script.copacetic.helper/script.py

```diff
@@ -1,18 +1,30 @@
 # author: realcopacetic
 
 from resources.lib.script.actions import *
-from resources.lib.utilities import clear_cache, sys
+from resources.lib.utilities import clear_cache, sys, urllib
 
 
 class Main:
     def __init__(self, *args):
         try:
             self.params = dict(arg.split('=', 1) for arg in args)
+            log(f'FUCK_IN: {self.params}', force=True)
```

Comment:
Could we rename this? We don't want little devs to read foul language 😄

Reply:
Ahh sorry haha, thought I had removed that, as you can probably tell I was getting quite frustrated with getting something to work - will remove it asap! Thanks for checking @basrieter

Reply:
OK, updated and squashed - thanks @basrieter